common/cache/domainCache.go

```diff
@@ -952,3 +955,10 @@ func (entry *DomainCacheEntry) IsSampledForLongerRetention(
 	}
 	return false
 }
+
+func (entry *DomainCacheEntry) HoursToAddToRetentionDuration() time.Duration {
+	if _, ok := entry.info.Data[JitteredWorkflowDeletionKey]; ok {
```

Comment:
we can make this more flexible/controllable:

instead of setting it to "true/false", let's add actual hours. This way we allow operator to define jitter period, 1 or 12 hours, for example.

If data provided is not a valid int, then use predefined default value